app/helpers/podcasts_helper.rb

```diff
@@ -29,4 +29,11 @@ def itunes_summary(model)
       sanitize_links_only(model.description || "")
     end
   end
+
+  def podcast_settings_active?
+    parts = request.path.split("/").select(&:present?)
+
+    # don't include podcast#show, or any episodes/feeds paths
+    parts[0] == "podcasts" && parts[2] && parts[2] != "episodes" && parts[2] != "feeds"
```

Comment:
The hierarchy of podcast routes doesn't _quite_ line up with when the `Settings` subnav link is active.  Have to filter for:

- `podcasts/123` the "podcast dashboard" thing
- `podcasts/123/episodes` listing episodes
- `podcasts/123/feeds` listing feeds